scaffold/cmd/templates/source/.github/workflows/test.yaml.tpl

```diff
@@ -0,0 +1,43 @@
+name: test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+    timeout-minutes: 30
+    name: "test"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v3
+        with:
+          go-version-file: go.mod
+          cache: true
+          cache-dependency-path: go.sum
+      - name: golangci-lint
+        uses: cloudquery/golangci-lint-action@master
```

Comment:
Let's use the official one? We only use our fork as caching is broken for the official one on monoreops